src/cli/domain/package-server-script/bundle/config/index.js

```diff
@@ -0,0 +1,63 @@
+/*jshint camelcase:false */
+'use strict';
+
+var webpack = require('webpack');
+var path = require('path');
+var wrapLoops = require('./wrapLoops');
+var externalDependenciesHandlers = require('./externalDependenciesHandlers');
+
+module.exports = function webpackConfigGenerator(params){
+  return {
+    entry: params.dataPath,
+    target: 'node',
+    output: {
+      path: '/build',
+      filename: params.fileName,
+      libraryTarget: 'commonjs2'
+    },
+    externals: externalDependenciesHandlers(params.dependencies),
+    module: {
+      loaders: [
+        {
+          test: /\.json$/,
+          exclude: /node_modules/,
+          loader: 'json-loader'
+        },
+        {
+          test: /\.js?$/,
+          exclude: /node_modules/,
+          loaders: [
+            'falafel-loader',
+            'babel-loader?' + JSON.stringify({
+              cacheDirectory: true,
+              'presets': [
+                [require.resolve('babel-preset-env'), {
+                  'targets': {
+                    'node': 0.10
+                  }
+                }]
+              ]
+            })
+          ],
+        }
+      ]
+    },
+    plugins: [
+      new webpack.optimize.OccurenceOrderPlugin(),
+      new webpack.optimize.UglifyJsPlugin({
+        compressor: {
+          warnings: false,
+          screw_ie8: true
+        },
+        sourceMap: false
```

Comment:
At the moment we are focusing on having a production grade-output only. Not sure we wanted to have sourceMaps outputted now. What you guys recon?